src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,17 @@
 package core.basesyntax;
 
+import lottery.Ball;
+import lottery.Lottery;
+
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery newLottery = new Lottery();
+        Ball[] ballsList = new Ball[3];
+        for (int i = 0; i < 3; ++i) {
+            ballsList[i] = newLottery.getRandomBall();
+        }
+        for (Ball ball : ballsList) {
+            System.out.println(ball.toString());
```

Comment:
```suggestion
            System.out.println(ball);
```
toString() is called automatically in System.out.println